src/main/webapp/index-script.js

```diff
@@ -499,23 +499,57 @@ function generateReview(review, currUser) {
   upvoteButton.innerHTML += '&#128077;' + review.positive;
   upvoteButton.id += "up" + review.id;
   upvoteButton.addEventListener("click", () => {
-    // Users must be logged in and can only vote once. 
-    if (currUser != null && !review.voters.includes(currUser)) {
-      review.positive += 1;
-      voteOnReview(review);
-      review.voters.push(currUser);
+    // Users must be logged in to vote! 
```

Comment:
decompose upvote/downvote into helper functon

Reply:
Yeah I was thinking about that as well - I'll work on decomposing after our pod sync!

Reply:
Can't be decomposed, when I do, it won't load the reviews at all.